docs/en/guides/pipeline-integrations/administration/advanced-toolkit-administration.md

```diff
@@ -0,0 +1,1526 @@
+new project. All settings, apps and folder configuration settings will be
+copied over to your new project. The following configurations were found:
+
+   big_buck_bunny: '/mnt/software/sgtk/big_buck_bunny'
+   ghosts: '/mnt/software/sgtk/ghosts'
+   chasing_perfection: '/mnt/software/sgtk/chasing_perfection'
+
+If you want to use any of the configs listed about for your new project, just
+type in its path when prompted below.
+
+You can use the Default Configuration for your new project.  The default
+configuration is a good sample config, demonstrating a typical basic setup of
+the {% include product %} Pipeline Toolkit using the latest apps and engines. This will be
+used by default if you just hit enter below.
+
+If you have a configuration stored somewhere on disk, you can just enter the
+path to this config it will be used for the new project.
+
+[tk-config-default]:
+Downloading Config tk-config-default v0.4.15 from the App Store...
+
+This is the 'Default Config' config.
+
+Below are all active projects, including ones that have been set up:
+--------------------------------------------------------------------
+
+[ 4] Demo Project
+     Fubar is an epic story of an on-going political wa...
+
+[ 5] Big Buck Bunny
+Note: This project has already been set up.
+     A killer bunny movie
+
+[ 6] The Ghosts of Pere Lachaise
+Note: This project has already been set up.
+     De Films en Aiguille and ChezEddy present this year...
+
+[ 7] Chasing Perfection
+Note: This project has already been set up.
+     You've seen the car commercials, the car races, and...
+
+[ 8] What Happened to My Headphones?
+     The Ryan Mayeda story
+
+Please type in the id of the project to connect to or ENTER to exit:
+```
+
+Toolkit provides you with a project name suggestion as part of the project setup process. If you are happy with the name, you can just press Enter to continue or alternatively type in another name by hand.
+
+If you have a special studio naming convention for projects, it is also possible to control the default value that the setup project process suggests. This is done via an advanced studio level hook - read more about it in the studio level hooks section further down in this document.
+
+### core
+
+Checks for Core API Updates. This will connect to the App Store and see if there is a more recent version of the Core API available. If there is, the command will ask you if you want to update.
+
+Example output from this command:
+
+```python
+> tank core
+
+Welcome to Sgtk!
+Starting Sgtk for your current directory '/private/tmp'
+- Running core...
+
+WARNING: You are potentially about to update the Core API for multiple
+projects.
+
+Welcome to the Sgtk update checker!
+This script will check if the Sgtk Core API
+installed in /mnt/software/sgtk/studio
+is up to date.
+
+You are currently running version v0.13.22 of the Sgtk Platform
+No need to update the Sgtk Core API at this time!
+Mannes-MacBook-Pro:tmp manne$
```

Comment:
This line should be removed

Reply:
Fixed in latest commit, thanks @mathurf !